### PR TITLE
Bump sdk version to 0.1.412

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Comment by Lir10** [comment]
@tzvatot ptal

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.412
+- [OCM-6763] Add default capability resource to SDK.
```
Comment (tzvatot) [inline CHANGES.md:7]: Please see v0.1.410 as example for bump

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (Lir10) [reply CHANGES.md]: done

**Review by tzvatot — APPROVED** [review]

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**